usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/basic/XWalkViewWithOnHideOnShowAsync.java

```diff
@@ -44,7 +44,7 @@ public final void onXWalkInitFailed() {
     public final void onXWalkInitCompleted() {
         StringBuffer mess = new StringBuffer();
         mess.append("Test Purpose: \n\n")
-        .append("Verifies XWalkView can hide and show.\n\n")
+        .append("Verifies XWalkView can hide and show.(Need to check this case in Android TV)\n\n")
```

Comment:
This case is not only for Android TV,  right?
How about this description:
Verifies XWalkView can hide and show. (such as on Android Mobile, Android TV)